### PR TITLE
OM - 307 | Terms of Service and register description

### PR DESCRIPTION
**Pull request by tuovinensanttu**

### CODE REVIEW

File: src/common/footer/Footer.module.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.content {
+    width: 1200px;
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+}
+
+@media (max-width: 1200px) {
```
Comment (jimppa) [inline src/common/footer/Footer.module.css]: Could you make this style mobile-first, i.e. make `width: auto;` the default for `.content` and override that when the screen is wide enough `@media (min-width: 1200px) { .content { width: 1200px; } }`?

File: src/common/footer/Footer.module.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    color: var(--hds-ui-color-black);
+    font-weight: bold;
+}
+
+.links:visited,
+.feedback:visited {
+    color: var(--hds-ui-color-black);
+}
+
+@media (max-width: 600px) {
```
Comment (jimppa) [inline src/common/footer/Footer.module.css]: Could you write the styles for `.textContainer` and `.feedback` in a mobile-first manner as well?

File: src/common/footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import { Link } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+
+import tosConstants from '../../pages/memberhsip/constants/tosConstants';
+import { ReactComponent as HelsinkiLogo } from '../svg/HelsinkiLogo.svg';
+import styles from './Footer.module.css';
+
+function Footer() {
+  const { t, i18n } = useTranslation();
+  const selectedLanguage =
```
Comment (jimppa) [inline src/common/footer/Footer.tsx]: I think you should use `i18n.languages[0]` to get the current language code ([source](https://www.i18next.com/overview/api#language))

File: src/i18n/fi.json [hunk 4]
```diff
@@ -88,6 +88,34 @@
     "approvedMessage": "Kiitos kun varmistit nuorisopalveluiden jäsenyyden.",
     "approvedLink": "Hakemus on jo hyväksytty / linkki on viallinen"
   },
+  "footer": {
+    "copyright": "Tekijänoikeus 2020",
```
Comment (jimppa) [inline src/i18n/fi.json]: I wonder if 2020 should instead be the current year from a date?

File: src/pages/memberhsip/components/termsOfService/TermsOfService.module.css [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.container {
+    width: 1200px;
+    margin: 20px auto 150px;
+    display: flex;
+    flex: 1 0 auto;
+ }
+
+@media (max-width: 1200px) {
```
Comment (jimppa) [inline src/pages/memberhsip/components/termsOfService/TermsOfService.module.css]: Could you write the styles for `.container` in a mobile-first manner?

File: src/pages/memberhsip/components/termsOfService/TermsOfService.module.css [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        margin: 20px 20px;
+    }
+}
+
+.back {
+    padding-top: 45px;
+    width: 20%;
+}
+
+@media (max-width: 767px) {
```
Comment (jimppa) [inline src/pages/memberhsip/components/termsOfService/TermsOfService.module.css]: Mobile-first here too

File: src/pages/memberhsip/components/termsOfService/TermsOfService.tsx [hunk 7]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```
Comment (jimppa) [inline src/pages/memberhsip/components/termsOfService/TermsOfService.tsx]: I think this component (and related files) could be in it's own directory, e.g. `src/pages/termsOfService`

File: src/common/footer/Footer.module.css (hunk 1 above)
Comment (tuovinensanttu) [reply src/common/footer/Footer.module.css]: Changed.

File: src/common/footer/Footer.module.css (hunk 2 above)
Comment (tuovinensanttu) [reply src/common/footer/Footer.module.css]: Changed

File: src/common/footer/Footer.tsx (hunk 3 above)
Comment (tuovinensanttu) [reply src/common/footer/Footer.tsx]: Yep, you are right. Changed.

File: src/i18n/fi.json (hunk 4 above)
Comment (tuovinensanttu) [reply src/i18n/fi.json]: Changed to current year.

File: src/pages/memberhsip/components/termsOfService/TermsOfService.module.css (hunk 6 above)
Comment (tuovinensanttu) [reply src/pages/memberhsip/components/termsOfService/TermsOfService.module.css]: Changed.

File: src/pages/memberhsip/components/termsOfService/TermsOfService.tsx (hunk 7 above)
Comment (tuovinensanttu) [reply src/pages/memberhsip/components/termsOfService/TermsOfService.tsx]: Changed.

**Pull request merged**